skops/card/tests/test_card.py

```diff
@@ -405,6 +428,85 @@ def test_with_metadata(self, card: Card, meth):
         assert result == expected
 
 
+class TestCardModelAttribute:
+    @pytest.fixture
+    def card(self):
+        model = LinearRegression(fit_intercept=False)
+        card = Card(model=model)
+        card.add(
+            model_description="A description",
+            model_card_authors="Jane Doe",
+        )
+        card.add_plot(
+            roc_curve="ROC_curve.png",
+            confusion_matrix="confusion_matrix.jpg",
+        )
+        card.add_table(search_results={"split": [1, 2, 3], "score": [4, 5, 6]})
+        return card
+
+    def path_to_card(self, path):
+        card = Card(model=path)
+        card.add(
+            model_description="A description",
+            model_card_authors="Jane Doe",
+        )
+        card.add_plot(
+            roc_curve="ROC_curve.png",
+            confusion_matrix="confusion_matrix.jpg",
+        )
+        card.add_table(search_results={"split": [1, 2, 3], "score": [4, 5, 6]})
+        return card
+
+    @pytest.mark.parametrize("meth", [repr, str])
+    @pytest.mark.parametrize("suffix", [".pkl", ".skops"])
+    def test_model_card_repr(self, card: Card, meth, suffix):
```

Comment:
I'm not sure if this test is strictly necessary. Maybe it is sufficient to show that calling `repr` or `str` on a card that has a path/str for model works? The fact that it's the same output is implied by the other tests.

Reply:
I changed it to only repr from card_from _path.